NightscoutUploadKit/DeviceStatus/LoopSuggested.swift

```diff
@@ -17,10 +17,9 @@ public struct LoopSuggested {
     let reason: String?
     let tick: Int?
     let correction: Double?
-    let predBGs: [Double]?
+    let predBGs: PredictedBG?
 
-
-    public init(timestamp: NSDate, rate: Double, duration: NSTimeInterval, eventualBG: Int, bg: Int, reason: String? = nil, tick: Int? = nil, correction: Double? = nil, predBGs: [Double]? = nil) {
+    public init(timestamp: NSDate, rate: Double, duration: NSTimeInterval, eventualBG: Int, bg: Int, reason: String? = nil, tick: Int? = nil, correction: Double? = nil, predBGs: PredictedBG? = nil) {
```

Comment:
What do you think about just having HKQuantity initializers, and dropping the double based ones?